tests/test_os_detection.py

```diff
@@ -112,74 +112,74 @@ def test_elf_parse_capa_pyinstaller_header():
     elf_header = zlib.decompress(
         b"".join(
             [
-                b"\x78\x9C\x8D\x56\x4F\x88\x1C\xD5\x13\xAE\x1D\x35\x0A\x7A\x58\x65",
```

Comment:
i wasn't expecting these changes. i dont think they're required for this PR, correct?

Reply:
typically I would discourage these changes from being including in the PR, but they are minor enough, no need to revert.